### PR TITLE
doc(hyprctl): enhance hyprctl help messages

### PR DESCRIPTION
**Pull request by memchr**
## Addtional Changes

- removed unused variables in hyprctl main.cpp
- prevent comparing integers of different signs

### CODE REVIEW

**Review by luxluth — COMMENTED** [review]
Looks Good but for me the dot at the end of a comment (`monitors:`) is to much cluter. Consider just writing `monitors`

File: hyprctl/main.cpp [hunk 1]
```diff
@@ -22,37 +22,54 @@
...
-    notify
-    globalshortcuts
-    instances
-
-flags:
+const std::string USAGE = R"#(usage:  hyprctl [flags] [<command> [args]]
+        hyprctl --batch {<command 1> [args] ; <command 2> [args] ; ...}
+
+LISTING COMMANDS:
+    monitors:           List outputs
```
Comment (vaxerski) [inline hyprctl/main.cpp:29]: why are the `:` everywhere?

File: hyprctl/main.cpp [hunk 2]
```diff
@@ -22,37 +22,54 @@
...
+    instances:          List running Hyprland instances
+    globalshortcuts:    List global shortcuts
+    version:            Print hyprland version
+
+CONFIGURATION COMMANDS:
+    keyword <keyword> [args]:   Execute a keyword
+    getoption <option>:         Get value of <option>
+    reload:                     Reload configurations
+
+PLUGIN:
```
Comment (vaxerski) [inline hyprctl/main.cpp:47]: plugins makes more sense

File: hyprctl/main.cpp [hunk 3]
```diff
@@ -22,37 +22,54 @@
...
+    plugin list:            List loaded plugins
+    plugin load <path>:     Load plugin from <path>
+    plugin unload <path>:   Unload plugin at <path>
+
+THEMING:
+    hyprpaper <keywords>        Issue hyprpaper keywords using IPC
```
Comment (vaxerski) [inline hyprctl/main.cpp:53]: hyprpaper should not be in theming, maybe a separate category called "integrations"?

File: hyprctl/main.cpp [hunk 4]
```diff
@@ -22,37 +22,54 @@
...
-    plugin
-    notify
-    globalshortcuts
-    instances
-
-flags:
+const std::string USAGE = R"#(usage:  hyprctl [flags] [<command> [args]]
+        hyprctl --batch {<command 1> [args] ; <command 2> [args] ; ...}
+
+LISTING COMMANDS:
```
Comment (vaxerski) [inline hyprctl/main.cpp:28]: query commands fits better imo

File: hyprctl/main.cpp (hunk 1 above)
Comment (BurningPho3nix) [reply hyprctl/main.cpp:29]: They probably took they style of writing from their language

**Review by BurningPho3nix — CHANGES_REQUESTED** [review]
If Vaxry's proposed changes are implemented, I would approve

**Pull request closed without merging**